### PR TITLE
fix: alert profile filter

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Fix alert profile filter used in `lacework alert-profile update`

This will return the correct list of profiles that can be updated. Non default profiles

### CODE REVIEW

File: cli/cmd/alert_profiles.go [hunk 1]
```diff
@@ -425,6 +425,21 @@ func promptAddAlertTemplate() api.AlertTemplate {
...
+	}
+
+	sort.Slice(profiles, func(i, j int) bool {
+		return profiles[i] < profiles[j]
+	})
+
+	return profiles
+}
+
 func filterAlertProfiles(response api.AlertProfilesResponse) []string {
```
Comment (afiune) [inline cli/cmd/alert_profiles.go:443]: Do we still need this function?

Comment (dmurray-lacework) [reply cli/cmd/alert_profiles.go:443]: yes

File: cli/cmd/alert_profiles.go [hunk 2]
```diff
@@ -425,6 +425,21 @@ func promptAddAlertTemplate() api.AlertTemplate {
 	}
 }
 
+func filterAlertProfilesByDefault(response api.AlertProfilesResponse) []string {
```
Comment (rmoles) [inline cli/cmd/alert_profiles.go:428]: `ByNonDefault` makes a bit more sense to me as we are filtering out the Default Profiles
```suggestion
func filterAlertProfilesByNonDefault(response api.AlertProfilesResponse) []string {
```

**Review by rmoles — APPROVED** [review]
I would prefer a different name for the function.
However this is a nitpick and doesn't change the functionality.
Happy to merge if you'd prefer to leave as is.

**Pull request merged**